components/timestamp/timestamp.tsx

```diff
@@ -142,7 +148,7 @@ class Timestamp extends PureComponent<Props, State> {
 
         // relative
         numeric: 'auto',
-        style: 'narrow',
+        style: 'long',
```

Comment:
Changing this default can affect other usages of this component, are we checking that in other places?

Reply:
Yes, though I don't think relative time is being used anywhere else yet. There's a contributor PR that is using it, but it's still in review and [specifies long](https://github.com/mattermost/mattermost-webapp/pull/5855/files/bfc2293620b64418f327462f10ecf7f52f7709b0#diff-035f90f92c7c4207c4bbca0cc40c402eR19).